python/foglamp/services/core/api/scheduler.py

```diff
@@ -27,9 +27,11 @@
     
     | GET POST        | /foglamp/schedule                                         |
     | GET PUT DELETE  | /foglamp/schedule/{schedule_id}                           |
+    | PUT             | /foglamp/schedule/{schedule_id}/enable                    |
+    | PUT             | /foglamp/schedule/{schedule_id}/disable                   |
```

Comment:
any documentation is available for these end points?

Reply:
I was thinking we would just use the current calls to modify a schedule to be enabled. Having these actions as well is not a problem however, I will add these to the documentation.